spec/factories/tag_field_factory.rb

```diff
@@ -2,6 +2,8 @@
 
 FactoryBot.define do
   factory :tag_field, class: DocumentType::TagField do
+    skip_create
```

Comment:
Waaah - this has been in the gem for 7 years! How did I not see this before? I love this :-).